apps/ledger-live-desktop/src/renderer/screens/manager/DeviceStorage/DeviceLanguageInstallation.tsx

```diff
@@ -1,114 +1,100 @@
-import React, { useState, useCallback, useEffect } from "react";
-import { Button, Flex, Icons, Drawer, Radio, BoxedIcon, Divider, Log } from "@ledgerhq/react-ui";
+import React, { useState, useCallback, useMemo } from "react";
+import { Button, Flex, Icons, Drawer, Radio, Divider, Text } from "@ledgerhq/react-ui";
 import { DeviceInfo, Language } from "@ledgerhq/types-live";
+import { track } from "~/renderer/analytics/segment";
```

Comment:
relative import should go after extern import, don't know why my linter is not complaining though 🤷

Reply:
I don't think `~/...` is considered to be a relative import

Reply:
ah that's probably why ! ok